src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
```

Comment:
It's better to name const variables using capital letters and underscores

Reply:
it really is